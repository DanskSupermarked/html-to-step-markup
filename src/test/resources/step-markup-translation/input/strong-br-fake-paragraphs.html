<div class="standardArticle_1"><p><strong>Lækkert havemøbelsæt i granit og stole i formstøbt plast</strong></p>  <br />  Havemøbelsæt bestående af 1 bord samt 6 stole i formstøbt plast.<br /> Vienne bordet er 100 x 205 cm og har et stel i galvaniseret stål og en bordplade med 6 granitplader.<br /> Iowa stolene er lavet i formstøbt plast, der ligner polyrattan-flet.<br /> Vienne sættet med 6 Iowa stole passer perfekt til den store familie, eller til par, som ofte har flere gæster<br /> Formstøbt plast skal blot tørres af med en klud med vand og sæbe, så hvis man foretrækker et minimum af vedligeholdelse, så er dette sæt det helt rigtige.<br /> <br /><br /> •	Granitsæt med 6 stole<br /> •	Stole i formstøbt plast<br /> •	Moderne og eksklusivt granitbord til hele familien på 100 x 205 cm<br /> •	Formstøbt plast giver et minimum af vedligeholdelse<br /> <br /><br /> Formstøbt plast:Det er et materiale, der ligner polyrattan-flet men dog er plastik.Dette gør det nemt at transportere og håndtere havemøblerne.<br /> <br /><br /> Granit: Granit er et meget holdbart materiale, som kan modstå al slags vejr. Granit er ikke menneskeskabt men derimod hugget direkte ud af bjerge i Fjernøsten, og derfor er to møbler aldrig ens. Struktur og farven vil variere fra bord til bord.<br /> <br /><br /> Vienne sæt med 6 Iowa stole er altså et lækkert havemøbelsæt til hele familien eller parret, der foretrækker et minimum af vedligeholdelse og et moderne, tidssvarende design.</div>
