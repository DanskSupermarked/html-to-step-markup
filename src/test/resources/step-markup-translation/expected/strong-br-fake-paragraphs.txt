<bold>Lækkert havemøbelsæt i granit og stole i formstøbt plast</bold><return/>
<return/>
Havemøbelsæt bestående af 1 bord samt 6 stole i formstøbt plast.<return/>
Vienne bordet er 100 x 205 cm og har et stel i galvaniseret stål og en bordplade med 6 granitplader.<return/>
Iowa stolene er lavet i formstøbt plast, der ligner polyrattan-flet.<return/>
Vienne sættet med 6 Iowa stole passer perfekt til den store familie, eller til par, som ofte har flere gæster<return/>
Formstøbt plast skal blot tørres af med en klud med vand og sæbe, så hvis man foretrækker et minimum af vedligeholdelse, så er dette sæt det helt rigtige.<return/>
<return/>
<bulletlist>
<bullet>Granitsæt med 6 stole</bullet>
<bullet>Stole i formstøbt plast</bullet>
<bullet>Moderne og eksklusivt granitbord til hele familien på 100 x 205 cm</bullet>
<bullet>Formstøbt plast giver et minimum af vedligeholdelse</bullet>
</bulletlist><return/>
Formstøbt plast:Det er et materiale, der ligner polyrattan-flet men dog er plastik.Dette gør det nemt at transportere og håndtere havemøblerne.<return/>
<return/>
Granit: Granit er et meget holdbart materiale, som kan modstå al slags vejr. Granit er ikke menneskeskabt men derimod hugget direkte ud af bjerge i Fjernøsten, og derfor er to møbler aldrig ens. Struktur og farven vil variere fra bord til bord.<return/>
<return/>
Vienne sæt med 6 Iowa stole er altså et lækkert havemøbelsæt til hele familien eller parret, der foretrækker et minimum af vedligeholdelse og et moderne, tidssvarende design.
